libraries/core_libs/consensus/src/transaction/transaction_manager.cpp

```diff
@@ -154,6 +155,13 @@ TransactionStatus TransactionManager::insertValidatedTransaction(std::shared_ptr
     proposable = false;
   }
 
+  if (kProposeDagBlockGasLimit < tx->getGas()) {
```

Comment:
you can use kConf.propose_dag_gas_limit here, no need to define this constant kProposeDagBlockGasLimit ?

Reply:
Done